dashboard/src/main/res/drawable-night/dashboard_ic_notification_bubble_badge.xml

```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="19dp"
+    android:height="20dp"
+    android:viewportWidth="19"
+    android:viewportHeight="20">
```

Comment:
The notification icons move a bit to the left when the bubble appears. It's probably a size issue.

Reply:
addressed.